x-pack/plugins/security_solution/public/common/hooks/eql/helpers.ts

```diff
@@ -0,0 +1,150 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+import moment from 'moment';
+import { Unit } from '@elastic/datemath';
+
+import { EqlSearchStrategyResponse } from '../../../../../data_enhanced/common';
+import { InspectResponse } from '../../../types';
+import { EqlPreviewResponse, Source } from './types';
+// eslint-disable-next-line @kbn/eslint/no-restricted-paths
+import { EqlSearchResponse } from '../../../../server/lib/types';
```

Comment:
Same comment as previous one, we should try to move this to the common section and not pull things from server within common if possible.

Reply:
Fixed! https://github.com/elastic/kibana/pull/78985/commits/ece38ef4fc2918b570731bf6c6a34784e992d5d8